<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_cestaSlozky">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>folder path</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_selectFolder">
        <property name="text">
         <string>vyber složku</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label_obrazek">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>500</width>
             <height>500</height>
            </size>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="3" column="1">
           <widget class="QLineEdit" name="lineEdit_path_3"/>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="pushButton_path_1">
            <property name="text">
             <string>Vybrat cestu</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit_path_0"/>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="pushButton_move_1">
            <property name="text">
             <string>Přesunout
F2</string>
            </property>
            <property name="shortcut">
             <string>F2</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="pushButton_move_0">
            <property name="text">
             <string>Přesunout
F1</string>
            </property>
            <property name="shortcut">
             <string>F1</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="pushButton_path_0">
            <property name="text">
             <string>Vybrat cestu</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="pushButton_move_3">
            <property name="text">
             <string>Přesunout
F4</string>
            </property>
            <property name="shortcut">
             <string>F4</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="lineEdit_path_4"/>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="pushButton_path_2">
            <property name="text">
             <string>Vybrat cestu</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QPushButton" name="pushButton_path_3">
            <property name="text">
             <string>Vybrat cestu</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QPushButton" name="pushButton_move_4">
            <property name="text">
             <string>Přesunout
F5</string>
            </property>
            <property name="shortcut">
             <string>F5</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="pushButton_move_2">
            <property name="text">
             <string>Přesunout
F3</string>
            </property>
            <property name="shortcut">
             <string>F3</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QPushButton" name="pushButton_path_4">
            <property name="text">
             <string>Vybrat cestu</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit_path_1"/>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEdit_path_2"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_fileName">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>FileName</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QPushButton" name="pushButton_back">
            <property name="text">
             <string>&lt;</string>
            </property>
            <property name="shortcut">
             <string>Left</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_forward">
            <property name="text">
             <string>&gt;</string>
            </property>
            <property name="shortcut">
             <string>Right</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_first">
            <property name="text">
             <string>první</string>
            </property>
            <property name="shortcut">
             <string>Home</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_last">
            <property name="text">
             <string>poslední</string>
            </property>
            <property name="shortcut">
             <string>End</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>877</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSoubor">
    <property name="title">
     <string>Soubor</string>
    </property>
    <addaction name="actionVyber_slo_ku"/>
    <addaction name="actionKonec"/>
   </widget>
   <addaction name="menuSoubor"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionVyber_slo_ku">
   <property name="text">
    <string>Vyber složku</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionKonec">
   <property name="text">
    <string>Konec</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
